pgmpy/estimators/expert.py

```diff
@@ -188,37 +225,58 @@ def estimate(
 
             selected_edge = nonedge_effects.iloc[nonedge_effects.effect.argmax()]
             edge_direction = None
-            if use_llm:
+
+            if orientations:
```

Comment:
Could you please add a comment here with "Step 3.2: ..."

Reply:
I think would help with readability if you could write the overall if-else flow in the comment.